opensoundscape/raven.py

```diff
@@ -212,13 +221,105 @@ def split_single_annotation(
             end,
             *list(pd.Series(species).isin(annots[col]).astype(int)),
         ]
-        splits_df = splits_df.append(segment_df)
+        splits_df = splits_df.append(segment_df, ignore_index=True)
+
+    return splits_df
+
+
+def split_single_annotation(
+    raven_file,
+    col,
+    split_len_s,
+    overlap_len_s=0,
+    total_len_s=None,
+    keep_final=False,
+    species=None,
+    min_label_len=0,
+):
+    """Split a Raven selection table into short annotations
```

Comment:
does this result in one-hot annotations for even-lengthed time segments? 

maybe could say "Aggregate one-hot annotations for even-lengthed time segments, drawing annotations from a specified column of a Raven selection table"

Reply:
Good suggestion; added your suggested text to the docstring